stress/basic.go

```diff
@@ -245,7 +245,26 @@ type BasicClient struct {
 	Concurrency   int      `toml:"concurrency"`
 	SSL           bool     `toml:"ssl"`
 	Format        string   `toml:"format"`
-	addrId        int
+
+	addrId   int
+	r        chan<- response
+	interval time.Duration
+}
+
+func (c *BasicClient) retry(b []byte, backoff time.Duration) {
+	bo := backoff + time.Duration(500*time.Millisecond)
```

Comment:
We tested with the linear increase and it worked well.  Did not see the need for the exponential increase.